### PR TITLE
[tests-only][full-ci] adding test to remove old versions of resources using cli command

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds test for removing versions via cli command

## Related Issue

- Coverage for: https://github.com/owncloud/ocis/issues/9472
- Part of https://github.com/owncloud/ocis/issues/9529

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/cliCommands/removeVersions.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Feature: remove versions via CLI command
+
+
+  Scenario: remove old versions of resources in storage
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "This is version 1" to "textfile.txt"
+    And user "Alice" has uploaded file with content "This is version 2" to "textfile.txt"
+    And user "Alice" has uploaded file with content "This is version 3" to "textfile.txt"
+    And the administrator has stopped the server
+    When the administrator removes all versions of resources using the CLI
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:11]:
```suggestion
    When the administrator removes all the file versions using the CLI
```

File: tests/acceptance/features/cliCommands/removeVersions.feature [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    Then the command should be successful
+    And the command output should contain "✅ Deleted 2 revisions (6 files / 2 blobs)"
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:13]:
```suggestion
    And the command output should contain "✅ Deleted 2 revisions (6 files / 2 blobs)"
    And <also we can check that file "textfile.txt" doesn't have any versions>
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/cliCommands/removeVersions.feature [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  Scenario: remove old versions of resources in storage
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "This is version 1" to "textfile.txt"
+    And user "Alice" has uploaded file with content "This is version 2" to "textfile.txt"
+    And user "Alice" has uploaded file with content "This is version 3" to "textfile.txt"
+    And the administrator has stopped the server
+    When the administrator removes all the file versions using the CLI
+    Then the command should be successful
+    And the command output should contain "✅ Deleted 2 revisions (6 files / 2 blobs)"
+    When user "Alice" gets the number of versions of file "textfile.txt"
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:14]: need to start the server again
```suggestion
    And the administrator has started the server
    When user "Alice" gets the number of versions of file "textfile.txt"
```

Comment (prashant-gurung899) [reply tests/acceptance/features/cliCommands/removeVersions.feature:14]: Nice.Got you!

File: tests/acceptance/features/cliCommands/removeVersions.feature [hunk 4]
```diff
@@ -0,0 +1,15 @@
+@env-config
+Feature: remove versions via CLI command
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:2]:
```suggestion
Feature: remove file versions via CLI command
```

File: tests/acceptance/features/cliCommands/removeVersions.feature [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+
+  Scenario: remove old versions of resources in storage
```
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:5]:
```suggestion
  Scenario: remove all file versions
```

File: tests/acceptance/features/cliCommands/removeVersions.feature
Comment (saw-jan) [inline tests/acceptance/features/cliCommands/removeVersions.feature:1]: Could you rename `removeVersions.feature` -> `removeFileVersions.feature`

**Review by saw-jan — COMMENTED** [review]
small comments

File: tests/acceptance/features/cliCommands/removeVersions.feature
Comment (prashant-gurung899) [reply tests/acceptance/features/cliCommands/removeVersions.feature:1]: > Could you rename `removeVersions.feature` -> `removeFileVersions.feature`

sure.

**Pull request merged**